arangod/Scheduler/SchedulerFeature.cpp

```diff
@@ -104,8 +104,16 @@ void SchedulerFeature::collectOptions(std::shared_ptr<options::ProgramOptions> o
                      new UInt64Parameter(&_nrMinimalThreads),
                      arangodb::options::makeDefaultFlags(arangodb::options::Flags::Hidden));
 
+    // max / min number of threads
+
+  // Concurrency throttling:
+  options->addOption("--server.ongoing-low-priority-multiplier", std::string("controls the number of low prio requests that can be ongoing at a given point in time, relative to the maximum number of request handling threads"),
+                     new DoubleParameter(&_ongoingLowPriorityMultiplier),
+                     arangodb::options::makeDefaultFlags(arangodb::options::Flags::Dynamic,
```

Comment:
I think the default value for this option is hard-coded, so the `Dynamic` attribute does not seem to be justified here.